added missing Ressources of CG:

patrick-werner opened the pull request:
MolecularDefinition

patrick-werner commented:
removed duplicated GenomicStudy entry

lmckenzi reviewed (commented):
Why did you move GenomicStudy to no longer be in alphabetic order?

patrick-werner commented:
> Why did you move GenomicStudy to no longer be in alphabetic order?

i introduced a duplicate by mistake and deleted the wrong entry. Fixed with latest commit
